myhoard/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.0.6-45-g2ce5877'
```

Comment:
This mustn't be committed. I don't know why this is not in .gitignore, though. I had already noticed and thought about this.

Reply:
Thanks for pointing that out. Removed the file and added it to the .gitignore.